bibliopixel/animation/circle.py

```diff
@@ -14,10 +15,10 @@ def __init__(self, layout, **kwds):
 
         self.rings = layout.rings
         self.ringCount = layout.ringCount
-        self.lastRing = layout.lastRing
+        if deprecated.allowed():
+            self.lastRing = layout.lastRing
```

Comment:
why is lastRing deprecated?

Reply:
Becauses it's nothing more or less than `ringCount - 1` - https://github.com/ManiacalLabs/BiblioPixel/blob/master/bibliopixel/layout/circle.py#L38-L41